build.gradle.kts

```diff
@@ -0,0 +1,29 @@
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+        maven("https://developer.huawei.com/repo/")
+        maven("https://maven.pkg.github.com/Rocket-Beer/*") {
+            println("**** Rocket Beer maven ****")
+
+            credentials {
+                val userName = publish.CommonMethods.getPublisherUserName(rootProject)
```

Comment:
this `publish.CommonMethods`is from the rocket plugin, no?

Reply:
Yeap, it is